README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Kodi version](https://img.shields.io/badge/kodi%20versions-18-blue)](https://kodi.tv/)
 [![GitHub release](https://img.shields.io/github/release/castagnait/plugin.video.netflix.svg)](https://github.com/castagnait/plugin.video.netflix/releases)
-[![Build Status](https://travis-ci.org/castagnait/plugin.video.netflix.svg?branch=master)](https://travis-ci.org/castagnait/plugin.video.netflix)
+[![CI](https://github.com/castagnait/plugin.video.netflix/workflows/CI/badge.svg)](https://github.com/castagnait/plugin.video.netflix/actions?query=workflow:CI)
```

Comment:
This have a weird path i don't have these addresses, previous ones have always worked

Reply:
It doesn't work because it is not merged yet. As soon as this is merged the URL will start working. See inputstreamhelper.

Reply:
Or the link in my comment above.